spec/requests/api/moves_controller_update_spec.rb

```diff
@@ -150,10 +150,10 @@
             }
           end
 
-          it 'updates the moves documents' do
-            expect(move.reload.documents).to match_array(before_documents)
+          it 'updates the moves profile documents' do
+            expect(move.profile.documents).to match_array(before_documents)
```

Comment:
do you think we need to test something before the action is done?
that means that we do not trust our setup

Reply:
I think it's more robust, yeah